sdk/python/jobs/finetuning/standalone/text-completion/ftaas_summarization_with_text_completion.ipynb

```diff
@@ -0,0 +1,407 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Text Completion - SamSum \n",
+    "\n",
+    "This sample shows how use `text-completion` components from the `azureml` system registry to fine tune a model to summarize a dialog between 2 people using samsum dataset. We then deploy the fine tuned model to an online endpoint for real time inference.\n",
```

Comment:
I think the notebook should be how to fine tune a model, not how to use components to do that since there are no components to choose